pkg/userop/testing/local_blockchain/account.go

```diff
@@ -0,0 +1,70 @@
+package local_blockchain
+
+import (
+	"bufio"
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"log/slog"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+type Account struct {
+	PrivateKey   *ecdsa.PrivateKey
+	Address      common.Address
+	TransactOpts *bind.TransactOpts
+}
+
+func NewAccount(ctx context.Context, node *EthNode) (Account, error) {
+	chainID, err := node.Client.ChainID(ctx)
+	if err != nil {
+		return Account{}, err
+	}
+
+	pvk, err := crypto.GenerateKey()
+	if err != nil {
+		return Account{}, fmt.Errorf("failed to generate deployer private key: %w", err)
+	}
+
+	opts, err := bind.NewKeyedTransactorWithChainID(pvk, chainID)
+	if err != nil {
+		return Account{}, err
+	}
+
+	return Account{
+		PrivateKey:   pvk,
+		Address:      crypto.PubkeyToAddress(pvk.PublicKey),
+		TransactOpts: opts,
+	}, nil
+}
+
+func NewAccountWithBalance(ctx context.Context, balance *big.Int, node *EthNode) (Account, error) {
```

Comment:
Add comment that balance passed as a parameter here is in ether, not wei or gwei.